gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Sat Dec 24 15:23:43 CET 2016
+#Sun Mar 12 10:04:45 CST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.2.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.4-all.zip
```

Comment:
I would prefer to keep dependency updates in separate PRs but it is fine. 
I want to merge the PR but there is a bug fix release 3.4.1, can you update it?